src/main/java/ch/njol/skript/conditions/IsServerWhitelisted.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.conditions;
+
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+@Name("Is Server Whitelisted")
+@Description("Whether or not the server is whitelisted.")
+@Examples("if server is whitelisted:")
+@Since("INSERT VERSION")
+public class IsServerWhitelisted extends Condition {
+	
+	static {
+		Skript.registerCondition(IsServerWhitelisted.class, "server (is|1¦is(n't| not)) white[ ]listed");
+	}
+	
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		setNegated(parseResult.mark != 1);
+		return true;
+	}
+	
+	@Override
+	public boolean check(Event e) {
+		return Bukkit.hasWhitelist() == isNegated();
+	}
+	
+	@Override
+	public String toString(@Nullable Event e, boolean debug) {
+		return "server is whitelisted";
```

Comment:
You should check if the condition is negated here, so if it is negated, you can return "server is not whitelisted"